og_sm_context/src/Plugin/OgGroupResolver/AdminGroupResolver.php

```diff
@@ -27,12 +28,12 @@ public function resolve(OgResolvedGroupCollectionInterface $collection) {
     if (!$route_object) {
       return;
     }
-    if (strpos($route_object->getPath(), '/group/node/{node}/') !== 0) {
+    if (!preg_match('#^/group/node/({node}|[0-9]+)#', $route_object->getPath())) {
```

Comment:
I don't like this change but it's the only way I see how we can remain site context for site routes.
Perhaps we should create a separate group-resolver for site routes?

Reply:
See dc2de11